README.md

```diff
@@ -60,130 +22,98 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 [CatalogOfMyThings] <a name="about-project"></a>
 
-> Describe your project in 1 or 2 sentences.
 
-**[your_project__name]** is a...
+**[CatalogOfMyThings]** is a catalog app which is designed with Ruby and you can add your books, games, music albums in that catalog.
 
 ## 🛠 Built With <a name="built-with"></a>
 
-### Tech Stack <a name="tech-stack"></a>
-
-> Describe the tech stack and include only the relevant sections that apply to your project.
-
-<details>
-  <summary>Client</summary>
-  <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
-  </ul>
-</details>
-
-<details>
-  <summary>Server</summary>
-  <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
-  </ul>
-</details>
-
-<details>
-<summary>Database</summary>
-  <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
-  </ul>
-</details>
+- **[Ruby]**
 
 <!-- Features -->
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
+- **[Presentation Video](https://drive.google.com/file/d/1alEWNXK7dtRjaZSo721sHGitvekmRx6K/view?usp=sharing)**
```

Comment:
The first code reviewer gave only two required changes to the readme file, and we made these changes.  The second code reviewer gave another required change to our presentation video. I know that required changes to make the video 3 person and shorter than 5 min but, our coding partner's brother reached us and told us our partner can't reach the internet and doesn't know when he can reach us. Because of this reason, we created a video with just 2 of us and explain our partner's part. When our partner reaches the internet we will create another video. And including the code reviewing policy, if the first reviewer didn't say anything about one situation, the second reviewer can give suggestions as optional about that situation. In these conditions can you please approve our project?
[6:12](https://microverse-students.slack.com/archives/D055F0HMSSK/p1683303142595959)